.devcontainer/devcontainer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "image": "sagebionetworks/genie:latest",
+    "image": "ghcr.io/sage-bionetworks/genie:develop",
```

Comment:
Don't think I've activated this before in VSCode development but modifying it here just in case